tests/resources/corsikaConfigTest.yml

```diff
@@ -9,36 +9,33 @@ primary: 'proton'
 
 # Energy range - [emin, emax]
 erange:
-- !astropy.units.Quantity
-  unit: !astropy.units.Unit {unit: GeV}
-  value: 0.01
-- !astropy.units.Quantity
-  unit: !astropy.units.Unit {unit: GeV}
-  value: 10.0
+  - unit: 'GeV'
+    value: 0.01
+  - unit: 'GeV'
+    value: 10.0
 
 # Slope of energy spectrum
 eslope: -2
 
 # Zenith angle - single value or list of len 2 for range with min and max zenith
-zenith: !astropy.units.Quantity
-  unit: !astropy.units.Unit {unit: deg}
+zenith:
+  unit: 'deg'
   value: 20.0
 
 # Azimuth angle - single value or list of len 2 for range with min and max azimuth
-phi: !astropy.units.Quantity
-  unit: !astropy.units.Unit {unit: deg}
+phi:
+  unit: 'deg'
   value: 0.0
 
 # CSCAT parameters - list of len 3
 # First entry is the number of uses of each shower
 # Second entry is the scattering radius in units of distance
 # Third entry is 0, for circlar scattering area
 cscat:
-- 10
-- !astropy.units.Quantity
-  unit: !astropy.units.Unit {unit: m}
-  value: 1500.0
-- 0
-viewcone: !astropy.units.Quantity
-  unit: !astropy.units.Unit {unit: deg}
+  - 10
```

Comment:
I am confused about this number 10 here without an accompanying key. (I know it was there before already)

Reply:
This is explained in the comment above:

```
# CSCAT parameters - list of len 3
# First entry is the number of uses of each shower
# Second entry is the scattering radius in units of distance
# Third entry is 0, for circlar scattering area
cscat:
  - 10
  - unit: 'm'
    value: 1500.0
  - 0
```

The early part of the code is full of these kind of 'choices'. Today we probably would require a value keyword for every entry. In this case, we only require a value key in case a parameter has a unit.

Reply:
Yes, a keyword for every value would be welcome, but ok, nothing for this PR.